spec/lib/vendor/serrano_spec.rb

```diff
@@ -102,13 +102,13 @@
 
     specify 'text2' do
       expect(src2.cached_string('text')).to eq(
-        'Kevan, D.K.M.E. & Wighton, D.C. (1981) Paleocene orthopteroids from south-central Alberta, Canada. Canadian Journal of Earth Sciences 18, 1824–1837.'
+        'Kevan, D.K.M.E. & Wighton, D.C. (1981) Paleocene orthopteroids from south-central Alberta, Canada. Canadian Journal of Earth Sciences, 18(12), 1824–1837. https://doi.org/10.1139%2Fe81-170'
       )
     end
 
     specify 'html1' do
-      expect(src1.cached_string('html')).to start_with(
-        'Brauer, A. (1909) <i>Die Süsswasserfauna Deutschlands. Eine Exkursionsfauna bearb. ... und hrsg. von Dr. Brauer.</i> G. Fischer, Available from: https://doi.org/10.5962%2Fbhl.title.1086')
+      expect(src1.cached_string('html')).to eq(
+        'Brauer, A. (1909) <i>Die Süsswasserfauna Deutschlands. Eine Exkursionsfauna bearb. ... und hrsg. von Dr. Brauer.</i> G. Fischer, https://doi.org/10.5962%2Fbhl.title.1086')
```

Comment:
Can we include `Available from`?

Reply:
Yes. Including DOI?

Reply:
Probably not for DOI?  Maybe just for URL or that other field (available from or something like it).

Reply:
updating URL filed. The other field is "howpublished". I checked all styles, non of them using this field, it is not even a valid variable for CSL. I checked what are valid values for the field (according to bibtex):
The howpublished field is used to store the a notice for unusual publications. Examples:
```
howpublished = "Distributed at the local tourist office"
howpublished = "WebPage"
howpublished = "Museum exhibit"
```

Reply:
The field which we are really missing in our bibtex is: "number-of-pages".  This is needed for book chapter to indicated the total number of pages in the book.